drivers/dma/dma_iproc_pax_common.h

```diff
@@ -0,0 +1,536 @@
+#define AE1_AE_FIFO_READ_POINTER 0x2120
+
+#define RM_RING_SECURITY_SETTING 0x3000
+#define RM_CONTROL 0x3008
+#define RM_TIMER_CONTROL_0 0x300c
+#define RM_TIMER_CONTROL_1 0x3010
+#define RM_BD_THRESHOLD 0x3014
+#define RM_BURST_LENGTH 0x3018
+#define RM_FIFO_FULL_THRESHOLD 0x301c
+#define RM_MASK_SEQUENCE_MAX_COUNT 0x3020
+#define AE_TIMEOUT 0x3024
+#define RING_OR_AE_STATUS_LOG_ENABLE 0x3028
+#define RING_FLUSH_TIMEOUT 0x302c
+#define RM_MEMORY_CONFIGURATION 0x3030
+#define RM_AXI_CONTROL 0x3034
+#define RM_GENERAL_MSI_DEVICE_ID 0x3038
+#define RM_GENERAL_MSI_ADDRESS_LS 0x303c
+#define RM_GENERAL_MSI_ADDRESS_MS 0x3040
+#define CONFIG_INTERRUPT_STATUS_MASK 0x3044
+#define CONFIG_INTERRUPT_STATUS_CLEAR 0x3048
+#define TOGGLE_INTERRUPT_STATUS_MASK 0x304c
+#define TOGGLE_INTERRUPT_STATUS_CLEAR 0x3050
+#define DDR_ADDR_GEN_INTERRUPT_STATUS_MASK 0x3054
+#define DDR_ADDR_GEN_INTERRUPT_STATUS_CLEAR 0x3058
+#define PACKET_ALIGNMENT_INTERRUPT_STATUS_MASK 0x305c
+#define PACKET_ALIGNMENT_INTERRUPT_STATUS_CLEAR 0x3060
+#define AE_INTERFACE_GROUP_0_INTERRUPT_MASK 0x3064
+#define AE_INTERFACE_GROUP_0_INTERRUPT_CLEAR 0x3068
+#define AE_INTERFACE_GROUP_1_INTERRUPT_MASK 0x306c
+#define AE_INTERFACE_GROUP_1_INTERRUPT_CLEAR 0x3070
+#define AE_INTERFACE_GROUP_2_INTERRUPT_MASK 0x3074
+#define AE_INTERFACE_GROUP_2_INTERRUPT_CLEAR 0x3078
+#define AE_INTERFACE_GROUP_3_INTERRUPT_MASK 0x307c
+#define AE_INTERFACE_GROUP_3_INTERRUPT_CLEAR 0x3080
+#define AE_INTERFACE_GROUP_4_INTERRUPT_MASK 0x3084
+#define AE_INTERFACE_GROUP_4_INTERRUPT_CLEAR 0x3088
+#define AE_INTERFACE_GROUP_5_INTERRUPT_MASK 0x308c
+#define AE_INTERFACE_GROUP_5_INTERRUPT_CLEAR 0x3090
+#define AE_INTERFACE_GROUP_6_INTERRUPT_MASK 0x3094
+#define AE_INTERFACE_GROUP_6_INTERRUPT_CLEAR 0x3098
+#define AE_INTERFACE_GROUP_7_INTERRUPT_MASK 0x309c
+#define AE_INTERFACE_GROUP_7_INTERRUPT_CLEAR 0x30a0
+#define AE_INTERFACE_TOP_INTERRUPT_STATUS_MASK 0x30a4
+#define AE_INTERFACE_TOP_INTERRUPT_STATUS_CLEAR 0x30a8
+#define REORDER_INTERRUPT_STATUS_MASK 0x30ac
+#define REORDER_INTERRUPT_STATUS_CLEAR 0x30b0
+#define DME_INTERRUPT_STATUS_MASK 0x30b4
+#define DME_INTERRUPT_STATUS_CLEAR 0x30b8
+#define REORDER_FIFO_PROG_THRESHOLD 0x30bc
+#define GROUP_PKT_EXTENSION_SUPPORT 0x30c0
+#define RM_GENERAL_MSI_DATA_VALUE 0x30c4
+#define RM_AXI_READ_BURST_THRESHOLD 0x30c8
+#define RM_GROUP_RING_COUNT 0x30cc
+#define MSI_INTERRUPT_STATUS_MASK 0x30d0
+#define MSI_INTERRUPT_STATUS_CLEAR 0x30d4
+#define RM_MSI_DISABLE 0x30d8
+#define RM_RESERVE 0x30fc
+#define RM_RING_FLUSH_STATUS 0x3100
+#define RING_SEQUENCE_NUMBER_OVERFLOW 0x3104
+#define AE_SEQUENCE_NUMBER_OVERFLOW 0x3108
+#define MAX_SEQUENCE_NUMBER_FOR_ANY_RING 0x310c
+#define MAX_SEQUENCE_NUMBER_ON_MONITOR_RING 0x3110
+#define MAX_SEQUENCE_NUMBER_ON_ANY_AE 0x3114
+#define MAX_SEQUENCE_NUMBER_ON_MONITOR_AE 0x3118
+#define MIN_MAX_LATENCY_MONITOR_RING_TOGGLE 0x311c
+#define MIN_MAX_LATENCY_MONITOR_RING_ADDRESSGEN 0x3120
+#define Ring_Activity 0x3124
+#define AE_Activity 0x3128
```

Comment:
Nit: Slightly odd to have a MixedCaps define in the middle of the rest.

Reply:
ack, will fix it.